src/azure-cli/azure/cli/command_modules/acs/_loadbalancer.py

```diff
@@ -6,11 +6,11 @@
 from distutils.version import StrictVersion  # pylint: disable=no-name-in-module,import-error
 
 # pylint: disable=no-name-in-module,import-error
-from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfile
-from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfileManagedOutboundIPs
-from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfileOutboundIPPrefixes
-from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfileOutboundIPs
-from azure.mgmt.containerservice.v2019_11_01.models import ResourceReference
+from azure.mgmt.containerservice.v2020_03_01.models import ManagedClusterLoadBalancerProfile
```

Comment:
for api version change, it requires full live test regression to avoid breaking change

Reply:
Manually tested all commands on my test cluster